### PR TITLE
DOCS-7173: CSM setup docs reorg

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

Restructures the CSM setup docs.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+code_lang: kubernetes
+type: multi-code-lang
+code_lang_weight: 60 # a number that represents relative weight. 
+---
+
+Use the following instructions to enable [CSM Threats][1] on Kubernetes. To learn more about the supported deployment types for each CSM feature, see [Setting Up Cloud Security Management][2].
+
+{{< tabs >}}
+
+{{% tab "Operator" %}}
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md:13]:
```suggestion
{{% tab "Datadog Operator" %}}
```

just for extra context

File: content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+1. Add the following to the `spec` section of the `values.yaml` file:
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md:15]:
```suggestion
1. Add the following to the `spec` section of the `datadog-agent.yaml` file:
```

the operator spec file is called datadog-agent.yaml

File: content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+    ```yaml
+    # values.yaml file
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md:18]:
```suggestion
    # datadog-agent.yaml file
```

File: content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+2. Restart the Agent.
+
+
+[2]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.v2alpha1.md
+
+{{% /tab %}}
+
+{{% tab "Helm" %}}
+
+1. Add the following to the `datadog` section of the `values.yaml` file:
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md:36]:
```suggestion
1. Add the following to the `datadog` section of the `datadog-values.yaml` file:
```

container docs generally refer to the custom helm config file as `datadog-values.yaml`—the `values.yaml` file is the default helm chart config, where parameters not set in `datadog-values.yaml` default to those in `values.yaml`

File: content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+    ```yaml
+    # values.yaml file
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md:39]:
```suggestion
    # datadog-values.yaml file
```

File: content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+      securityAgent:
+        runtime:
+          enabled: true
+    ```
+
+2. Restart the Agent.
+
+{{% /tab %}}
+
+{{% tab "Daemonset" %}}
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md:52]:
```suggestion
{{% tab "DaemonSet" %}}
```

File: content/en/security/cloud_security_management/setup/csm_enterprise/_index.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+The Cloud Security Management (CSM) Enterprise package includes [CSM Threats][1], [CSM Misconfigurations][2] (cloud accounts and Agent), [CSM Identity Risks][3], and [CSM Vulnerabilities][4] (container images and hosts). To learn more about the available CSM packages, see [Setting up Cloud Security Management][8].
+
+## Getting started
+
+To enable CSM Enterprise on your infrastructure, complete the following steps:
+
+### Enable resource scanning for cloud accounts
+
+To enable resource scanning for your cloud accounts, you must first set up the integration and then enable CSM for each AWS account, Azure subscription, and Google Cloud project. For detailed instructions, see [Enable CSM Enterprise for Cloud Accounts][22].
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_enterprise/_index.md:33]:
```suggestion
To enable resource scanning for your cloud accounts, you must first set up the integration and then enable CSM for each AWS account, Azure subscription, or Google Cloud project. For detailed instructions, see [Enable CSM Enterprise for Cloud Accounts][22].
```

File: content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+code_lang: kubernetes
+type: multi-code-lang
+code_lang_weight: 60 # a number that represents relative weight. 
+---
+
+Use the following instructions to enable [CSM Misconfigurations][1], [CSM Threats][2], and [CSM Vulnerabilities][3] on Kubernetes. To learn more about the supported deployment types for each CSM feature, see [Setting Up Cloud Security Management][4].
+
+{{< tabs >}}
+
+{{% tab "Operator" %}}
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md:13]:
```suggestion
{{% tab "Datadog Operator" %}}
```

File: content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+1. Add the following to the `spec` section of the `values.yaml` file:
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md:15]:
```suggestion
1. Add the following to the `spec` section of the `datadog-agent.yaml` file:
```

File: content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+
+    ```yaml
+    # values.yaml file
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md:18]:
```suggestion
    # datadog-agent.yaml file
```

File: content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+2. Restart the Agent.
+
+[2]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.v2alpha1.md
+
+{{% /tab %}}
+
+{{% tab "Helm" %}}
+
+1. Add the following to the `datadog` section of the `values.yaml` file:
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md:51]:
```suggestion
1. Add the following to the `datadog` section of the `datadog-values.yaml` file:
```

File: content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+
+    ```yaml
+    # values.yaml file
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md:54]:
```suggestion
    # datadog-values.yaml file
```

File: content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+          enabled: true
+        host:
+          enabled: true
+    ```
+
+2. Restart the Agent.
+
+{{% /tab %}}
+
+{{% tab "Daemonset" %}}
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_enterprise/agent/kubernetes.md:81]:
```suggestion
{{% tab "DaemonSet" %}}
```

File: content/en/security/cloud_security_management/setup/csm_pro/agent/kubernetes.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+code_lang_weight: 60 # a number that represents relative weight. 
+---
+
+Use the following instructions to enable container image metadata collection and [Software Bill of Materials (SBOM)][1] collection in the Datadog Agent for [CSM Vulnerabilities][2]. This allows you to scan the libraries in container images to detect vulnerabilities. Vulnerabilities are evaluated and and scanned against your containers every hour.
+
+To learn more about the supported deployment types for each CSM feature, see [Setting Up Cloud Security Management][3].
+
+{{< tabs >}}
+
+{{% tab "Operator" %}}
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_pro/agent/kubernetes.md:15]:
```suggestion
{{% tab "Datadog Operator" %}}
```

File: content/en/security/cloud_security_management/setup/csm_pro/agent/kubernetes.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+
+1. Add the following to the `spec` section of the `values.yaml` file:
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_pro/agent/kubernetes.md:17]:
```suggestion
1. Add the following to the `spec` section of the `datadog-agent.yaml` file:
```

File: content/en/security/cloud_security_management/setup/csm_pro/agent/kubernetes.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+            enabled: true
+    ```
+
+2. Restart the Agent.
+
+{{% /tab %}}
+
+{{% tab "Helm" %}}
+
+1. Add the following to the `datadog` section of the `values.yaml` file:
```
Comment (cswatt) [inline content/en/security/cloud_security_management/setup/csm_pro/agent/kubernetes.md:40]:
```suggestion
1. Add the following to the `datadog` section of the `datadog-values.yaml` file:
```

**Review by cswatt — CHANGES_REQUESTED** [review]
mostly just changing the names of the config files for k8s install with operator and helm to what's used in the containers docs

File: content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md (hunk 4 above)
Comment (drichards-87) [reply content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md:36]: Didn't realize that. That's good to know! Thanks for the heads up.

File: content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+    # datadog-agent.yaml file
+    spec:
+      features:
+        remoteConfiguration:
+          enabled: true
+        cws:
+          enabled: true
+    ```
+
+2. Restart the Agent.
```
Comment (FouadWahabi) [inline content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md:27]: I think technically it's more correct to say `Apply the changes` then restart the agent in the case of k8s Operator installation

**Review by cswatt — APPROVED** [review]

**Pull request merged**